### PR TITLE
Added user input prior to script running

### PR DESCRIPTION
**Pull request by jwu910**
Script should require user input to determine how the script should proceed.

cc: @rainrivas  @phuchle @AlChiu @psyduc

### CODE REVIEW

**Comment by jwu910** [comment]
I did not get rid of your trash commit. it somehow got merged in to my branch lol
@rainrivas

File: README.md [hunk 1]
```diff
@@ -6,3 +6,5 @@ This script will combine an overlay image with a second image, save a copy, and
 ### Technologies used:
 - watchdog
 - Python Imaging Library (PIL)
+
+**Trash commit line**
```
Comment (rainrivas) [inline README.md]: you should delete this trash commit line if we don't need it here

Comment (jwu910) [reply README.md]: Can I just remove it in the code, or do we not want it in history? lol

Comment (jwu910) [reply README.md]: Trash commit removed.

**Comment by rainrivas** [comment]
I think it was initially my commit.  Not sure.  Anyway, code looks good to me

**Review by rainrivas — APPROVED** [review]

**Pull request merged**